homeassistant/components/reolink/host.py

```diff
@@ -273,6 +273,15 @@ async def register_webhook(self) -> None:
         webhook_path = webhook.async_generate_path(event_id)
         self._webhook_url = f"{base_url}{webhook_path}"
 
+        if base_url.startswith("https"):
+            _LOGGER.warning(
+                "Reolink products can not push motion events to a HTTPS address (SSL), "
+                "please configure a local HTTP address under "
+                "settings->system->network->Home Assistant URL->local network, "
+                "current address: %s",
+                self._webhook_url,
+            )
```

Comment:
Couple of things:

- These kinds of things should be raised as repairs instead.
- You could test this during integration setup/integration setup, to check if a local URL is available (instead of relying on the resulting webhook URL)

Reply:
@frenck thank you for the feedback!
I have converted it to a repairs issue, bare with me because it is my first time implementing a repairs issue.

Actually the `register_webhook` will only be called once (under normal circomstances) during the integration setup.
async_init->self.subscribe->self.register_webhook